go/tasks/plugins/webapi/agent/plugin.go

```diff
@@ -101,6 +103,10 @@ func (p Plugin) Create(ctx context.Context, taskCtx webapi.TaskExecutionContextR
 		return nil, nil, err
 	}
 
+	if taskTemplate.GetContainer() != nil {
+		taskTemplate.GetContainer().Args = argTemplate
```

Comment:
qq: why do we need to revert args here? did you see any error in the propeller?

Reply:
For tasks with the same task function, the first render will persist, causing all prefixes to correspond to the same node. The {{}} templates will have been replaced so later renders will not find anything to replace.

Reply:
This solution was chosen over makeing a copy of the entire taskTemplate. I'm not familiar with if there is a better place to solve this problem.

Reply:
LGTM, could you add a comment at line 105?

Reply:
Done.

Reply:
thanks!